src/components/Forms/workspace/workspaceForm.tsx

```diff
@@ -32,6 +32,13 @@ const useStyles = makeStyles((theme: Theme) =>
 		},
 		button: {
 			color: theme.palette.common.white,
+			marginLeft: theme.spacing(1),
+		},
+		cancelButton: {
```

Comment:
You have specfified static color on hover. How does it works with other themes?

Reply:
its for text on button which will be always on primary color its works fine on both themes